### PR TITLE
New release of GeneralizedMetropolisHastings for Julia 0.5

### PR DESCRIPTION
**Pull request by KrisDM**
New version which is compatible with Julia 0.5 (while backwards compatible with 0.4)
Minor bug fixes and improvements

### CODE REVIEW

**Comment by andreasnoack** [comment]
You should consider https://github.com/attobot/attobot

File: GeneralizedMetropolisHastings/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,8 @@
+julia 0.4.5
+Compat 0.9.5
```
Comment (tkelman) [inline GeneralizedMetropolisHastings/versions/0.1.0/requires]: are you actually requiring a feature that was first released in 0.9.5? minimum version requirements do not mean "this is the version I tested with," that's very likely to be too tight and rule out versions that do actually work

Comment (KrisDM) [inline GeneralizedMetropolisHastings/versions/0.1.0/requires]: Thanks for the suggestion. Good point. I jumped without checking from 0.8.6 (which didn't have what I needed) to 0.9.5 (at that moment latest release). On a general note, how could I find out for sure without having to test each release in turn?

**Comment by KrisDM** [comment]
@andreasnoack Thanks, I installed it, will use it next time.

File: GeneralizedMetropolisHastings/versions/0.1.0/requires (hunk 1 above)
Comment (tkelman) [inline GeneralizedMetropolisHastings/versions/0.1.0/requires]: best way is checking blame on the readme for the features you're using

Comment (KrisDM) [inline GeneralizedMetropolisHastings/versions/0.1.0/requires]: Thanks, that worked. I'll bump it down again and use attabot to release a patch.

**Pull request merged**